solana/programs/wormhole-governance/src/instructions/governance.rs

```diff
@@ -9,7 +9,7 @@
 //! instruction is able to invoke the program's admin instructions.
 //!
 //! The instruction needs to be encoded in the VAA payload, with all the
-//! accounts. These accounts may be in any order, with two placeholder accounts:
+//! accounts. These accounts may be in any order and must include two additional placeholder accounts:
```

Comment:
these are actually optional, so maybe 
```suggestion
//! accounts. These accounts may be in any order and may include two additional placeholder accounts:
```